sources/shared/Stride.NuGetResolver/NuGetAssemblyResolver.cs

```diff
@@ -255,7 +257,12 @@ private static void RegisterNativeDependencies(Dictionary<string, string> assemb
         var registerDependencyMethod = nativeLibraryHelperType.GetMethod("RegisterDependency")
             ?? throw new InvalidOperationException($"Couldn't find method 'RegisterDependency' in {nativeLibraryHelperType}");
         foreach (var lib in nativeLibs)
+        {
+#if DEBUG
+            Console.WriteLine($"Register native library: {lib}");
+#endif
```

Comment:
What's this one about ?

Reply:
Some debug logs because there are still issues on Linux with some native libraries like Silk.NET.Sdl.